src/components/NavigationBar/MainPageNavBarStyled.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+import SmallLogo from '@/images/small_logo.svg';
+
+export const NavBarLayout = styled.nav`
+${(props) =>
+    props.type === 'header'
```

Comment:
Magic string.. https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings

Reply:
renamed to navBarType.